content/e/ppw/2023-02-04-ppw-back-to-the-og.md

```diff
@@ -24,7 +24,7 @@ source = "[Official PpW Facebook]"
   - c: "[PpW Championship](@/c/ppw-championship.md)"
     s: "Hardcore Match"
 - credits:
-    Host, Ring Announcer: '["OG" Michael HT](@/w/michael-ht.md)'
+    Host, Ring Announcer: '[Michael HT](@/w/michael-ht.md)'
```

Comment:
While it reduces the number of people in the list, this was a nice bit of flavor. I'd keep it.